src/syscon.rs

```diff
@@ -70,8 +70,14 @@ impl SYSCON {
 
             bod: BOD(PhantomData),
             flash: FLASH(PhantomData),
+            #[cfg(feature = "82x")]
             irc: IRC(PhantomData),
+            #[cfg(feature = "82x")]
             ircout: IRCOUT(PhantomData),
+            #[cfg(feature = "845")]
+            fro: FRO(PhantomData),
+            #[cfg(feature = "845")]
+            froout: FROOUT(PhantomData),
```

Comment:
I think it would be better if the code here were the same for both platforms. That would require us to settle on one name (see other comment) and hide the difference in a type definition.